### PR TITLE
Audit

### PR DESCRIPTION
**Pull request by dvcorreia1984**
In this pull request the following was done: 

1. Many rows were added to populate and test database performance
2. Performance was improved by indexing the tables

Screenshot 2
Screenshot 3
screenshot 4
Screenshot 5
screenshot 6
Screenshot 7

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -61,3 +61,8 @@ CREATE TABLE visits (
     CONSTRAINT pk_visits PRIMARY KEY (animals_id, vets_id, date_of_visit)
 );
 
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+   create index animal_id_desc on visits(animal_id desc);
+   create index vet_id_desc on visits(vet_id desc);
+   create index email_desc on owners(email desc);
```
Comment (tugsem) [inline schema.sql]: [ OPTIONAL ]
This is a nice-to-have a suggestion as it won't improve the query that was reported as problematic in the requirements for this project.


As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:


>CREATE INDEX ON owners ((lower(email)));

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello team 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️indexes created succesfully
✔️screenshots added
✔️professional documentation
## Required Changes ♻️

-  [ ] Please include your data.sql file with the INSERT INTO statement you have used for this milestone.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (dvcorreia1984) [reply schema.sql]: Thank you @tugsem

**Pull request merged**